packages/react-components/lib/date-time-picker.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Yeah, I thought a native solution will be better for the browser support, but I guess I was wrong. =0

Reply:
Had this error with Moment lib, fixed by downgrading `date-io` to 1.3.13 

```
utils.getYearText is not a function
TypeError: utils.getYearText is not a function
    at DateTimePickerToolbar (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:269584:18)
    at renderWithHooks (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:220318:18)
    at mountIndeterminateComponent (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:222997:13)
    at beginWork (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:224111:16)
    at HTMLUnknownElement.callCallback (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:205703:14)
    at Object.invokeGuardedCallbackDev (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:205752:16)
    at invokeGuardedCallback (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:205807:31)
    at beginWork$1 (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:228718:7)
    at performUnitOfWork (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:227672:12)
    at workLoopSync (http://localhost:6006/vendors~main.e6775aeedff598925967.bundle.js:227645:22)
```

It's a known bug. 
https://github.com/mui-org/material-ui-pickers/issues/1442


Having this warning on running test:
`
 Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.`

This is on purpose because I'm removing the timezone to make those tests.

Reply:
>  This is on purpose because I'm removing the timezone to make those tests.

Can' we test with timezones? Is there a way to tell jsdom what locale settings to use?

Reply:
Fixed moment warning by moving the date to ISO format.